services/requester/requester.go

```diff
@@ -323,6 +323,10 @@ func (e *EVM) Call(
 	from common.Address,
 	height uint64,
 ) ([]byte, error) {
+	e.logger.Debug().
```

Comment:
Moved this to the top of the method, so that we always log the call data. Previously, there were 2 possible paths that we could return an error, before logging the call data.